packages/fxa-auth-server/test/scripts/convert-customers-to-stripe-automatic-tax.ts

```diff
@@ -463,13 +472,45 @@ describe('StripeAutomaticTaxConverter', () => {
         });
       });
 
+      describe("invalid IP address, isn't in same country", () => {
+        beforeEach(async () => {
+          geodbStub.returns({
+            postalCode: 'ABC',
+            countryCode: 'ZZZ',
+          });
+
+          stripeHelperStub.currencyHelper.isCurrencyCompatibleWithCountry.returns(
+            false
+          );
+
+          result = await stripeAutomaticTaxConverter.enableTaxForCustomer({
+            ...mockCustomer,
+            metadata: {
+              userid: mockIpAddressMapping[0].uid,
+            },
+          });
+        });
+
+        it('does not update customer', () => {
+          expect(updateStub.notCalled).true;
+        });
+
+        it('returns false', () => {
+          expect(result).false;
+        });
+      });
+
       describe('valid IP address', () => {
         beforeEach(async () => {
           geodbStub.returns({
             countryCode: 'US',
             postalCode: 92841,
           });
 
+          stripeHelperStub.currencyHelper.isCurrencyCompatibleWithCountry.returns(
```

Comment:
`Property 'returns' does not exist on type '(currency:string | null | undefined, country: string | null | undefined) => boolean'. ts(2339) [Ln510, Col 75]`

Fix?:
```
stripeHelperStub.currencyHelper.isCurrencyCompatibleWithCountry = sinon.stub().returns(
            true
          );
```

Reply:
Yep!